doc/quickstart_authentication.rst

```diff
@@ -13,23 +13,22 @@ And use it as credentials in your management configuration client. These three i
 
 * OAuth authentication using Active Directory application and service principal
 * OAuth authentication using Azure Active Directory user/password
-* A wrapper on top of `ADAL for Python <https://github.com/AzureAD/azure-activedirectory-library-for-python>`
+* A wrapper on top of `ADAL for Python <https://github.com/AzureAD/azure-activedirectory-library-for-python>`__
 
 Using Service Principal
 ------------------------
 
-There is now a detailed official tutorial to describe this:
-https://azure.microsoft.com/en-us/documentation/articles/resource-group-create-service-principal-portal/
+To set up Service Principal authentication, you must have:
 
-At this point, you must have:
+* Your client ID, referred to in the Azure portal as the application ID.
+* A secret authentication key for the application.
+* Your Active Directory tenant id,
+  referred to in the Azure portal as the directory ID.
 
-* Your client id. Found in the "client id" box in the "Settings" page of your application in the Azure portal.
-* Your secret key. Generated when you have created the application. You cannot show the key after creation.
-  If you've lost the current key, you must create a new one in the "Settings" page of your application.
-* Your AD tenant id. It's an UUID (e.g. ABCDEFAB-1234-ABCD-1234-ABCDEFABCDEF) which points to the AD containing your application.
-  You will find it in the URL when you are in the Azure portal in your AD, or in the "view endpoints" in any of the given url.
+For help finding these values in the Azure portal, please refer to
+`this tutorial <https://azure.microsoft.com/en-us/documentation/articles/resource-group-create-service-principal-portal/>`__.
```

Comment:
Please rename the link to include the content. Instead of "this tutorial" => "the Service Principal authentication tutorial"

Please remove "en-us" from the Url, so it will be localized according to the browser. This is a general comment for all doc on microsoft.com